charts/radarr/values.yaml

```diff
@@ -1,151 +1,11 @@
 # Default values for radarr.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
 
-image:
-  repository: linuxserver/radarr
-  tag: 3.0.0.3543-ls21
-  pullPolicy: IfNotPresent
-
-# upgrade strategy type (e.g. Recreate or RollingUpdate)
-strategyType: Recreate
-
-# Probes configuration
-probes:
-  liveness:
-    initialDelaySeconds: 60
-    failureThreshold: 5
-    timeoutSeconds: 10
-  readiness:
-    initialDelaySeconds: 60
-    failureThreshold: 5
-    timeoutSeconds: 10
-
-# Prometheus Metrics
-exportarr:
-  enabled: false
+radarr:
   image:
-    repository: onedr0p/exportarr
-    tag: v0.3.0
+    organization: itscontained
```

Comment:
Why do we switch from linuxserver/radarr to itscontained/radarr here? The linuxserver images are multi-arch and much wider used.